en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+
+   @[youtube](https://youtu.be/3fNJQraCJx0)
+
+   @[youtube](https://youtu.be/3en4DlQF4XU)
+
+*Because it’s cumbersome to insert the wires once the top plate is assembled, do the wiring beforehand. 
+Although the design is well built such that you can do this later as well.
+ 
+### Power
+
+- The Pixhawk 6C gets powered by a power module PM02 (in this case).
+- This power module is supplied by a battery (4S 16.8V 5200 mAh)
+- The motors get power through the power distribution board.
+
+![motors_pdb_pixhawk6c](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/motors_pdb_pixhawk6c.png)
+
+
+### Arms
+
+**Screw-** Socket Cap Screw M3*38 16pcs | Flange Locknut M3 16pcs
+
+@[youtube](https://youtu.be/66Hfy6ysOpg)
+
+1. Putting the arms is quite simple as the motors come pre-assembled.  - Ensure that you have the right numbered arm with its motor on the respective side.
+
+   @[youtube](https://youtu.be/45KCey3WiJ4)
+
+   :::tip
+   Use your allen keys/ any elongated item and insert it on the opposite side of the bolt that you're trying to fasten.
+   :::
+
+1. Take one arm and insert the rectangle extrusion inside the rectangular hollow on the bottom plate.
+
+   @[youtube](https://youtu.be/GOTqmjq9_3s)
+
+1. While inserting the top plate on top of this the 3 piece assembly (bottom plate, top plate & arms) have to screwed using Socket Cap Screw M3*38 and Flange Locknut M3. 
+1. Hold one side using the mini cross wrench provided in the developer kit.
+
+   @[youtube](https://youtu.be/2rcNVekJQd0)
+
+1. Do not fasten any screws before all 3 motors are in place as this might make it difficult while you’re assembling the 3rd and 4th motor.
+
+   @[youtube](https://youtu.be/SlKRuNoE_AY)
+
+### Propellers
+
+@[youtube](https://youtu.be/V7JXI5-qrAs)
+
+- The bottom plate indicates the direction of the motor.
+- The propellers that have a white/silver coating go on that respective motor with the similar coat.
+- The unlocking and locking of the propeller is indicated on the propeller itself.
+- Use the 4 propellers and insert them on the motors keeping the above 3 points in mind.
+
+*The below parts can be placed as per user
+
+### GPS
+
+**Screw-** Locknut M3 4 pcs | Screw M3*10 4pcs
+
+@[youtube](https://youtu.be/aiFxVJFjlos)
+
+- Assemble the GPS by following the video.
+- This guide uses the GPS mount location according to the suggestion in Holybro’s guide.
+- Screw the GPS mount’s bottom end on the payload holder side using Locknut M3 & Screw M3*10
+
+@[youtube](https://youtu.be/uG5UKy3FrIc)
+
+### Pixhawk 6C
+
+- The wire from the PM02 goes to POWER1 in Pixhawk
+- The telemetry goes to TELEM1
+- The GPS to GPS1
+
+@[youtube](https://youtu.be/wFlr_I3jERQ)
+
+### Companion Computer (Optional)
+
+**Screw-** Socket Cap Screw M2.5*12 4pcs | Nylon Standoff M2.5*5 4pcs Locknut M2.5 4pcs
+
+- The X500 kit is awesome because of it’s design consideration of most common hardware used on drones.
```

Comment:
This is confusing English and has nothing to do with Companion computers.

Reply:
@akshata-01 I have no idea what this means. I'll make something up, but you will need to create a new PR to fix it after this merges.